helper/src/main/java/com/hazelcast/examples/helper/HazelcastUtils.java

```diff
@@ -19,11 +19,11 @@
 import com.hazelcast.cluster.Cluster;
 import com.hazelcast.cluster.Member;
 import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.instance.impl.HazelcastInstanceImpl;
```

Comment:
Again, reference to `internal`/`impl` API

Reply:
Done